setup.py

```diff
@@ -292,6 +292,8 @@ def build_deps():
     report('-- Building version ' + version)
 
     def check_file(f):
+        if bool(os.getenv("USE_SYSTEM_LIBS", False)):
+            return
```

Comment:
Are you suuuuure? :)

Reply:
Sure. As per Debian policy, the submodules under the third_party directory should not exist when building the package... So none of them exist. And we have to skip the checks in that case.